matrix/recognition/v1/recognition_service.proto

```diff
@@ -15,15 +15,15 @@
 
 syntax = "proto3";
 
-package matrixlabs.vision.v1;
+package matrix.recognition.v1;
 
-import "matrixlabs/vision/v1/vision_service.proto";
-import "matrixlabs/vision/v1/recognition.proto";
+import "matrix/vision/v1/vision_service.proto";
```

Comment:
I split recognition from vision as they are basically different services. `vision.proto` contains message definitions while `vision_service.proto` declarse vision service calls. The same applies to the recognition service.